layouts/blog/list.html

```diff
@@ -6,8 +6,8 @@
 <div class="relative max-w-7xl mx-auto px-4 sm:px-2 md:px-10 lg:px-2">
     <div class="mt-9 sm:mt-11 lg:mt-14">
         <ul class="grid sm:grid-cols-2 lg:grid-cols-3 gap-6 md:gap-8">
-            {{ $paginator := .Paginate .Data.Pages }}
-            {{ range $paginator.Pages }}
+            {{ $pages := where .Site.RegularPages "Section" "blog" }}
+            {{ range (.Paginate $pages 12).Pages }}
```

Comment:
The default value of 10 gave the appearance of an almost empty row. Increasing to 12 will ensure that whether the grid has 3 columns of 2 columns, that a row won't look empty.